slycot/tests/test.py

```diff
@@ -64,6 +64,111 @@ def test_td04ad_static(self):
             nr,a,b,c,d = transform.td04ad(rc,nin,nout,index,den,num)
 
 
+# ===================================================
+# Begin tb05ad tests
+import numpy as np
+
+from numpy.testing import assert_raises, assert_almost_equal
+from numpy.testing import assert_array_almost_equal, dec
+
+CASES = {}
+
+CASES['fail1'] = {'A': np.array([[-0.5,  0.,  0.,  0. ],
+                              [ 0., -1.,  0. ,  0. ],
+                              [ 1.,  0., -0.5,  0. ],
+                              [ 0.,  1.,  0., -1. ]]),
+                   'B': np.array([[ 1.,  0.],
+                             [ 0.,  1.],
+                             [ 0.,  0.],
+                             [ 0.,  0.]]),
+                   'C': np.array([[ 0.,  1.,  1.,  0.],
+                             [ 0.,  1.,  0.,  1.],
+                             [ 0.,  1.,  1.,  1.]])}
+
+n = 20
+p = 10
+m = 14
+np.random.seed(40)
+
+CASES['pass1'] = {'A': np.random.rand(n, n),
+                  'B':  np.random.rand(n, m),
+                  'C':  np.random.rand(p, n)}
+
+
+class test_tb05ad(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_tb05ad_ng(self):
+        for key in CASES:
```

Comment:
No problem. I had read on the [nump site](https://github.com/numpy/numpy/blob/master/doc/TESTS.rst.txt#writing-your-own-tests) that test functions shouldn't have docstrings. I thought that was odd.